code/sha3/Hacl.Impl.SHA3.Vec.fst

```diff
@@ -984,6 +984,7 @@ let get_multiblock_t (m:m_spec{is_supported m}) =
 inline_for_extraction noextract
 val get_multiblock_1: #m:m_spec{lanes m == 1} -> get_multiblock_t m
 
+#push-options "--max_ifuel 1 --max_fuel 1"
```

Comment:
fuel is for unrolling definitions in the smt solver; if you know that both of these are necessary, then the recommended style is to use `--fuel 1 --ifuel 1` to avoid first trying with 0/0, then failing, then trying with 1/1

Reply:
I replaced it with more proper options, my observation is that it's more reliable to use basic arithmetics of FStar than the ones in `Lib.IntTypes` e.g. for `get_multiblock_1` function we can use `size (v i * v rateInBytes)` instead of `i *! rateInBytes` that makes the verification process more stable and faster even when assisting the resolver with the following assert `assert (v (i *! rateInBytes) == v i * v rateInBytes)` and that could go for other operations where possible.